client/src/components/Navbar.js

```diff
@@ -0,0 +1,136 @@
+import React, { Fragment } from "react";
+import { Link } from "react-router-dom";
+import {
+  AppBar,
+  Toolbar,
+  List,
+  ListItem,
+  ListItemText,
+  Avatar,
+  withWidth, 
+  Menu, 
+  MenuItem, 
+  IconButton
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+
+const useStyles = makeStyles({
+  navDisplay: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  linkText: {
+    textDecoration: "none",
+    color: "#222222",
+    fontWeight: 700,
+    marginRight: 5,
+  },
+  navbar: {
+    backgroundColor: "#EEEEEE",
+  },
+});
+
+function getNavigation({ width, classes, anchorEl, handleClick, handleClose }) {
+  const navLinks = [
+    {
+      title: "Become A Sitter",
+      path: "/become-a-sitter",
+    },
+    {
+      title: "My Sitters",
+      path: "/my-sitters",
+    },
+    {
+      title: "My Jobs",
+      path: "/my-jobs",
+    },
+    {
+      title: "Messages",
+      path: "/messages",
+    },
+  ];
+  if (width === "sm" || width === "xs") {
+    return (
+      <Fragment>
+        <IconButton
+          aria-label="more"
+          aria-controls="long-menu"
+          aria-haspopup="true"
+          onClick={handleClick}
+        >
+          <MoreVertIcon />
+        </IconButton>
+        <Menu
+          id="menu"
+          anchorEl={anchorEl}
+          keepMounted
+          open={Boolean(anchorEl)}
+          onClose={handleClose}
+        >
+          {navLinks.map(({ title, path }) => (
+            <Link className={classes.linkText} to={path} key={title}>
+              <MenuItem onClick={handleClose}>{title}</MenuItem>
+            </Link>
```

Comment:
ReactRouter has a [NavLink](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/NavLink.md) component that would work well for this. It lets you apply styling to the link based on whether "to"  matches the current url

Reply:
Updated!